app/actions/UploadInventory.js

```diff
@@ -67,6 +67,10 @@ const changeStatusAndUpload = async (response, oneInventory, dispatch) => {
                   )
                     .then(() => resolve())
                     .catch((err) => {
+                      dbLog.error({
+                        logType: LogTypes.INVENTORY,
+                        message: `Failed to change inventory status: ${SYNCED} with inventory id: ${oneInventory.inventory_id}. Error at: /action/upload/changeInventoryStatus, -> ${err}`,
+                      });
```

Comment:
This will throw an error and might crash the app since err is an `object` type and `message` property in realm is a string type

Reply:
If this is a problem here, maybe other usages of `dbLog` should also be checked?

Reply:
Why not use `logStack: JSON.stringify(err)` as used elsewhere for the err object?

Reply:
I scanned the codebase and add some missing `logStack` params for `dbLog.error` in https://github.com/Plant-for-the-Planet-org/treemapper/pull/404 (this one not included of course!)

Reply:
Yes i have asked Tejas to do the same.